### ISSUE TITLE
spec 0.1: error using dpkg.py

### ISSUE THREAD

**Issue by simleo** [CellMigStandOrg/biotracks#5]
Using the `9I5TT808_F00000010.ome.xml` and  `cell_track_dpkg.ini` provided on the gdrive share.

```
[simleo:dpkg] (spec_0.1) % python3 dpkg.py 9I5TT808_F00000010.ome.xml 
/usr/local/lib/python3.5/site-packages/IPython/html.py:14: ShimWarning: The `IPython.html` package has been deprecated since IPython 4.0. You should import from `notebook` instead. `IPython.html.widgets` has moved to `ipywidgets`.
  "`IPython.html.widgets` has moved to `ipywidgets`.", ShimWarning)
>>>
Reading a TrackMate XML file version 3.4.2
>>> Found 4648 unique spots
>>> Found 4117 unique edges
Traceback (most recent call last):
  File "dpkg.py", line 21, in <module>
    dict_ = readfile.read_file(f)
  File "/Users/simleo/git/cell_track_dpkg/dpkg/readfile.py", line 243, in read_file
    (objects, links) = read_trackMate(f)
  File "/Users/simleo/git/cell_track_dpkg/dpkg/readfile.py", line 139, in read_trackMate
    if index == 0 and (tmp.iloc[index].SPOT_TARGET_ID) != (tmp.iloc[index + 1].SPOT_SOURCE_ID):
  File "/usr/local/lib/python3.5/site-packages/pandas/core/indexing.py", line 1312, in __getitem__
    return self._getitem_axis(key, axis=0)
  File "/usr/local/lib/python3.5/site-packages/pandas/core/indexing.py", line 1628, in _getitem_axis
    self._is_valid_integer(key, axis)
  File "/usr/local/lib/python3.5/site-packages/pandas/core/indexing.py", line 1542, in _is_valid_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds
```

**Comment by pcmasuzzo** [comment]
I cannot replicate this bug. 

I encountered it a few days ago as well, but fixed with commit 2252dc0ff4a3b4ef5485c6ad094c81877d8e5192.

The bug (exception) comes from tracks consisting only of two spots (source and target). 
Can you try again with a fresh pull of the branch? Thanks.

**Comment by simleo** [comment]
After updating pandas to 0.19.2 and reinstalling the above bug went away. I ran into a `FileNotFoundError` instead, but it should be fixed by #7.

**Comment by simleo** [comment]
Reopening. Can be closed when #7 is merged.

**Comment by pcmasuzzo** [comment]
#7 is merged.
@simleo feel free to close the issue if everything is fine.
